{"name": "test2_login_with_error_password", "status": "passed", "attachments": [{"name": "stdout", "source": "5b807e65-1507-4a59-a896-fa62f59e7f16-attachment.txt", "type": "text/plain"}], "start": 1691400032139, "stop": 1691400048146, "uuid": "2239f591-e3a1-4a3a-a6a8-9785e0484cd6", "historyId": "a59fe43aa9d3514c6555f810b0694901", "testCaseId": "a59fe43aa9d3514c6555f810b0694901", "fullName": "testcase.login_logout.test_login_with_error_info.TestErrorLogin#test2_login_with_error_password", "labels": [{"name": "feature", "value": "Login error"}, {"name": "severity", "value": "minor"}, {"name": "story", "value": "login with error password"}, {"name": "parentSuite", "value": "testcase.login_logout"}, {"name": "suite", "value": "test_login_with_error_info"}, {"name": "subSuite", "value": "TestErrorLogin"}, {"name": "host", "value": "Joys-MacBook-Pro.local"}, {"name": "thread", "value": "6750-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.login_logout.test_login_with_error_info"}]}